### PR TITLE
Splits organs into organ/internal

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
I hate myself.

The bulk of the changes, for reviewers, is in code/modules/organs/internal/_internal.dm.

This also adds BP_ defines to use instead of strings.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
changes to vueui are not intentional but i can't spot them

**Comment by NonQueueingMatt** [comment]
!wip

**Comment by NonQueueingMatt** [comment]
The bulk of the changes, for reviewers, is in code/modules/organs/internal/_internal.dm

**Comment by NonQueueingMatt** [comment]
!review

File: code/__defines/chemistry.dm [hunk 1]
```diff
@@ -48,7 +48,8 @@
 #define CE_STABLE "stable" // Inaprovaline
 #define CE_ANTIBIOTIC "antibiotic" // Spaceacilin
 #define CE_BLOODRESTORE "bloodrestore" // Iron/nutriment
-#define CE_PAINKILLER "painkiller"
+#define CE_PAINKILLER "painkiller" //General toxins
```
Comment (Geevies) [inline code/__defines/chemistry.dm]:
```suggestion
#define CE_PAINKILLER "painkiller"
```

File: code/__defines/chemistry.dm [hunk 2]
```diff
@@ -48,7 +48,8 @@
...
+#define CE_TOXIN "toxins"
```
Comment (Geevies) [inline code/__defines/chemistry.dm]:
```suggestion
#define CE_TOXIN "toxins" //General toxins
```

**Comment by NonQueueingMatt** [comment]
nvm found the vueui changes

File: code/controllers/subsystems/records.dm [hunk 3]
```diff
@@ -209,7 +209,7 @@
 				department = 1
 				if ((depthead || rank == "Captain") && manifest[positionType].len != 1)
 					manifest[positionType].Swap(1, manifest[positionType].len)
-				if(positionType == "head")
+				if(positionType == BP_HEAD)
```
Comment (NonQueueingMatt) [inline code/controllers/subsystems/records.dm]:
```suggestion
				if(positionType == BP_HEAD)
```
```suggestion
				if(positionType == "head")
```

File: code/game/objects/items/devices/lighting/flashlight.dm [hunk 4]
```diff
@@ -65,7 +65,7 @@
 			if(H.species.vision_organ)
 				vision = H.internal_organs_by_name[H.species.vision_organ]
 			if(!vision)
-				to_chat(user, span("warning", "You can't find any [H.species.vision_organ ? H.species.vision_organ : "eyes"] on [H]!"))
+				to_chat(user, span("warning", "You can't find any [H.species.vision_organ ? H.species.vision_organ : BP_EYES] on [H]!"))
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/lighting/flashlight.dm]:
```suggestion
				to_chat(user, span("warning", "You can't find any [H.species.vision_organ ? H.species.vision_organ : "eyes"] on [H]!"))
```

**Pull request merged**